### ISSUE TITLE
Timeout value is not evaluated in testQuality and AudioOnly

### ISSUE THREAD

**Issue by enricop89** [opentok/opentok-network-test-js#127]
Hi,

There is a bug when you set a custom timeout in the OTNetworkTest() constructor for an audio-only test. The user can set a value between 10s and 30s, but the code will always take 10s (because testTimeout for audio-only is 10s).

https://github.com/opentok/opentok-network-test-js/blob/e8c5a0d87070239b4aeb5c27a2bad04f8a9c09f4/src/NetworkTest/testQuality/index.ts#L308

Steps to reproduce:

```
var options = { audioOnly: true, timeout: 15000};
const otNetworkTest = new NetworkTest(OT, sessionInfo, options);
otNetworkTest.testQuality();
```

**Comment by jeffswartz** [comment]
Good catch, @enricop89. I am wondering if we should just update the docs to say that the maximum length of an audio-only tests is 10 seconds. 10 seconds should be enough time to determine audio quality.... Or have you encountered conditions where more time would be appropriate?

**Comment by enricop89** [comment]
@jeffswartz I agree on update the docs. In audio-only cases, I would say 10s are enough
